website/docs/r/nsxt_ip_set.html.markdown

```diff
@@ -42,11 +51,12 @@ The following arguments are supported:
 
 * `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful
   when connected as sysadmin working across different organisations.
-* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level.
+* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level. **Deprecated**
```

Comment:
I suggest we state deprecations first thing here (but leave the explanation at the end of the sentence for more info):

```suggestion
* `vdc` - (Deprecated; Optional) The name of VDC to use, optional if defined at provider level. **Deprecated**
```

We did this in https://registry.terraform.io/providers/vmware/vcd/latest/docs with the old maxRetryTimeout field.

CC: @mikeletux @adambarreiro @Didainius

Reply:
added